data/themes/darktable.css

```diff
@@ -1258,6 +1257,106 @@ filechooser row:hover,
   border: 0px;
 }
 
+/*--------------------------------
+  - module layouts manager dialog -
+  --------------------------------*/
+
+#modulegroups_editor
+{
+  color: @fg_color;
+  background-color: @plugin_bg_color;
+  padding-left: 0.5em;
+}
+#modulegroups-iop-header,
+#modulegroups-header-center,
+#modulegroups-preset,
+#modulegroups-preset-activated
+{
+  background-color: @bg_color;
+  color: @plugin_label_color;
+  padding: 0.5em 0.5em 0.5em 0.5em;
+  margin: 0 0 0.2em 0;
```

Comment:
Not consistent with preferences. It would be better to make them consistent and add them on existing preferences settings (around lines 1160 in preferences) would be better and avoid multiply lines on CSS.

Reply:
I've tried to do that in last commit... but not all can be shared as widgets structure is not the same. I hope it's ok for you !